Assets/HoloToolkit/UX/Resources/Models/BoundingBoxes/BoundingBoxes.fbx.meta

```diff
@@ -0,0 +1,230 @@
+fileFormatVersion: 2
+guid: 9a6f31c448d9f2245b211502c19c54db
+timeCreated: 1450214136
+licenseType: Pro
+ModelImporter:
+  serializedVersion: 22
+  fileIDToRecycleName:
+    100000: //RootNode
+    100002: lc_line02
+    100004: lc_line03
+    100006: lc_line04
+    100008: lc_line05
+    100010: lc_line06
+    100012: lc_line07
+    100014: lc_line08
+    100016: lc_line09
+    100018: lc_line10
+    100020: lc_line11
+    100022: lc_line12
+    100024: pCube2
```

Comment:
What's with the naming of these assets?

Reply:
No idea. This is a modified version of an MRTK asset, I'm not sure who created it initially.